From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Wed, 9 Jan 2019 14:54:39 +0000
Subject: [PATCH] pgp: Fix emu build and tests after pgp module renaming

Commit b07feb8746c3bb845e3f0d33d37c0bded704d14d (verifiers: Rename
verify module to pgp module) renamed the "verify" module to "pgp", but
the GRUB_MOD_INIT and GRUB_MOD_FINI macros were left as "verify", which
broke the emu target build; and file_filter_test still referred to the
now non-existent "verify" module. Fix both of these.

Signed-off-by: Colin Watson <cjwatson@ubuntu.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
(cherry picked from commit ed087f0460516737e174222f01e2bf6ccbd45674)
Signed-off-by: Daniel Axtens <dja@axtens.net>
---
 grub-core/commands/pgp.c  | 4 ++--
 tests/file_filter_test.in | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/grub-core/commands/pgp.c b/grub-core/commands/pgp.c
index 29e74a640..5c913c2e2 100644
--- a/grub-core/commands/pgp.c
+++ b/grub-core/commands/pgp.c
@@ -950,7 +950,7 @@ struct grub_file_verifier grub_pubkey_verifier =
 static grub_extcmd_t cmd, cmd_trust;
 static grub_command_t cmd_distrust, cmd_list;
 
-GRUB_MOD_INIT(verify)
+GRUB_MOD_INIT(pgp)
 {
   const char *val;
   struct grub_module_header *header;
@@ -1009,7 +1009,7 @@ GRUB_MOD_INIT(verify)
   grub_verifier_register (&grub_pubkey_verifier);
 }
 
-GRUB_MOD_FINI(verify)
+GRUB_MOD_FINI(pgp)
 {
   grub_verifier_unregister (&grub_pubkey_verifier);
   grub_unregister_extcmd (cmd);
diff --git a/tests/file_filter_test.in b/tests/file_filter_test.in
index bfb638227..ed6abcb5a 100644
--- a/tests/file_filter_test.in
+++ b/tests/file_filter_test.in
@@ -19,7 +19,7 @@ grubshell=@builddir@/grub-shell
 
 . "@builddir@/grub-core/modinfo.sh"
 
-filters="gzio xzio lzopio verify"
+filters="gzio xzio lzopio pgp"
 modules="cat mpi"
 
 for mod in $(cut -d ' ' -f 2 "@builddir@/grub-core/crypto.lst"  | sort -u); do
